src/app/api/boosts/claim/route.ts

```diff
@@ -0,0 +1,130 @@
+import { createClient } from '@supabase/supabase-js';
+import { NextResponse, type NextRequest } from 'next/server';
+import '@/utils/helper';
+import { Network, Alchemy, TokenBalance } from "alchemy-sdk";
+import { hoursToMilliseconds } from 'date-fns';
+
+const settings = {
+    apiKey: process.env.ALCHEMY_ID,
+    network: Network.BASE_MAINNET,
+};
+
+const alchemy = new Alchemy(settings);
+
+async function ownsNFT(userAddress: string, contract: string) {
+    const response = await alchemy.nft.verifyNftOwnership(userAddress, contract);
```

Comment:
@amit-shah-cb is using Alchemy SDK the best way to check whether a user owns NFT?

Isn't this a simple check onchain? probably minor optimization

Reply:
we can stick to alchemy for now, later can transition to ethers.Contract and rpc directly